### ISSUE TITLE
Final AverNet ckpt from google drive only obtain 25.67 psnr on davis_t6

### ISSUE THREAD

**Issue by KlZOnmyway** [XLearning-SCU/2024-NeurIPS-AverNet#4]
Hi, thanks for providing the pretrained model. I tried to run inference on davis_t6 testset, but only achieved a relatively low average PSNR 25.6781. Is there by any chance the uploaded ckpt is not the final weight but an intermediate one?

<img width="1472" alt="Image" src="https://github.com/user-attachments/assets/9adc6799-55b9-4b7b-b919-39e02d5427fa" />

**Comment by Pandint** [comment]
Hi!

I tested the uploaded model on DAVIS_t6 and obtained a PSNR of over 34.30.

Could you confirm that no other modifications were made on your end? Also, would you be able to share the option file you used for testing?

**Comment by KlZOnmyway** [comment]
Hi the config is as below, and there is nothing changed to the video_recurrent_model.py

```
name: AverNet_davis # The name of experiments
model_type: VideoRecurrentModel
scale: 1
num_gpu: 1  # set num_gpu: 0 for cpu mode
manual_seed: 0

datasets:
  test:
    name: TUD
    type: VideoRecurrentTestDataset
    dataroot_gt: {
        
        'davis': '/home/user/aio_restoration/dataset/davis_t6/gt',
      }
    dataroot_lq: {
       
        'davis': '/home/user/aio_restoration/dataset/davis_t6/blur',
      }

    cache_data: false
    io_backend:
      type: disk

    num_frame: -1

# network structures
network_g:
  type: AverNet
  mid_channels: 96
  prompt_dim: 96
  num_blocks: 7
  spynet_pretrained: experiments/pretrained_models/flownet/spynet_sintel_final-3d2a1287.pth

# path
path:
  pretrain_network_g: /home/user/aio_restoration/dataset/AverNet_final.pth
  strict_load_g: true

# validation settings
val:
  save_img: true
  suffix: ~  # add suffix to saved images, if None, use exp name
  patch_testing_size: 0
  overlap_size: 20

  metrics:
    psnr: # metric name, can be arbitrary
      type: psnr
    ssim:
      type: ssim
```

**Comment by Pandint** [comment]
Hi! Thanks for sharing the option file. I noticed that the test dataset path points to davis_t6/blur. Could you confirm whether you modified the test set compared to the original setting?

Additionally, if you aim for better generalization on purely blurred data, you may need to fine-tune on only blurry data or include such data in the training set and train from scratch. The pretrained model was only trained on data with time-varying degradations, so it may not generalize optimally to purely blurred inputs.

**Comment by KlZOnmyway** [comment]
Hi, thanks for further reply. I downloaded the test set (t6 davis) from the google drive link you provided. Blur is just a placeholder name.

**Comment by Pandint** [comment]
Hi!

Thanks for your clarification. We have tested the uploaded model on multiple devices, and the results consistently exceed 34.30 PSNR. We did not observe any performance degradation.

Could you verify your testing environment and settings again? For further debugging, you might consider running the test on a different machine or checking for any potential issues with dependencies.
